packages/components/psammead-timestamp/src/index.jsx

```diff
@@ -15,17 +15,22 @@ const StyledTimestamp = styled.time`
   color: ${C_CLOUD_DARK};
   display: block;
   font-family: ${GEL_FF_REITH_SANS};
-  padding-bottom: ${GEL_SPACING_HLF};
-  &:last-child {
-    padding-bottom: ${GEL_SPACING_DBL};
-  }
-`;
+  ${props =>
+    props.padding &&
+    css`
```

Comment:
This a bit tricky to parse, can you pull this out into a variable and then use it here please